samples/CustomSender.AspNetCore/project.json

```diff
@@ -0,0 +1,53 @@
+{

```

Comment:
BTW if this app doesn't need users and membership, could start out with a much simpler template.